src/edu/wright/cs/raiderplanner/controller/ChatController.java

```diff
@@ -64,6 +64,7 @@ public static void createUserMessagePane() {
 		userMessagePane.add(spacingBox, 1, 0);
 		userMessagePane.add(sendButton, 2, 0);
 		sendButton.setMinWidth(100);
+        sendButton.setDefaultButton(true);
```

Comment:
The indentation here is incorrect and you used spaces instead of tabs.  CheckStyle in Eclipse flags this.